packages/zowe-explorer/src/configuration/Constants.ts

```diff
@@ -76,6 +76,7 @@ export class Constants {
     public static SETTINGS_DS_DEFAULT_PDS = "zowe.ds.default.pds";
     public static SETTINGS_DS_DEFAULT_EXTENDED = "zowe.ds.default.extended";
     public static SETTINGS_DS_DEFAULT_PS = "zowe.ds.default.ps";
+    public static readonly SETTINGS_DS_TEMPLATES = "zowe.ds.templates";
```

Comment:
Should we mark other constants in this file as readonly too?

Reply:
we can, I got flagged for my addition by sonarcloud.